spec/integrations/shared/base_render_examples.rb

```diff
@@ -50,6 +50,33 @@
     it('returns json with a renamed field') { should eq(result) }
   end
 
+  context 'Given blueprint has ::fields with :discard_nil argument' do
+    context 'Given :discard_nil argument is true' do
+      let(:result) { '{"first_name":"Meg"}' }
+      let(:blueprint) do
+        Class.new(Blueprinter::Base) do
+          field :first_name, discard_nil: true
+          field :my_field, discard_nil: true do
+            nil
+          end
```

Comment:
**suggestion**: Can we alter this test so that the value on the provided `object` is `nil`, instead of explicitly set as part of the `Blueprint` itself (primarily to mirror a real-world scenario as best as possible).

Reply:
@lessthanjacob Yes we could, but that would mean moving the spec outside of base_render_exemples to a place like base_spec because blueprints are defined per test it seem

Reply:
@lessthanjacob I moved the spec but i'm not sure it really is better because the `base_spec` is more about specific scenario like "is rails context" or "is hash", or .... `exclude_if_nil` is something that applies to everything. I made sure it would be easy to revert by making a different commit in case I was wrong